www/js/app.js

```diff
@@ -1561,6 +1582,29 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
 
     // A string to hold any anchor parameter in clicked ZIM URLs (as we must strip these to find the article in the ZIM)
     var anchorParameter;
+    
+    /**
+     * Warn the user that he/she clicked on an external link, and open it in a new tab
+     * 
+     * @param {Event} event the click event (on an anchor) to handle
+     */
+    function warnAndOpenExternalLinkInNewTab(event) {
+        event.preventDefault();
+        event.stopPropagation();
+        var target = event.target.target;
+        var message = 'Do you want to open this external link?';
+        if (!target || target === '_blank') {
+            message += ' (in a new tab)';
+        }
+        message += '<br />' + event.target.href + '<br />(this behavior can be disabled in settings if you know what you do)';
+        uiUtil.systemAlert(message, 'Opening external link', true).then(function (response) {
```

Comment:
Instead of `<br />`, I would try wrapping each part of this message in `<p>` tags. At the moment it looks crowded and a bit ugly. The URL should be on its own line, with some vertical whitespace between the URL and the surrounding messages.

Reply:
I agree, will use `<p>` tags